### ISSUE TITLE
Workday API Clients should have `Non-expiring refresh tokens` enabled

### ISSUE THREAD

**Issue by bryanyap** [workato/docs#595]
Not enabling this will result in refresh tokens becoming invalid after awhile, causing the connection in Workato to fail silently.

![screen shot 2018-09-14 at 12 51 13 pm](https://user-images.githubusercontent.com/5821001/45530379-e11a6a80-b81c-11e8-8dc1-7be6563adaea.png)

**Comment by eeshansim** [comment]
@bryanyap Expiration time of refresh tokens are refreshed everytime an access token is refreshed.

Refresh tokens in client apps that are not marked as "Non-expiring Refresh Tokens" will only expire (become invalid) if there is prolonged inactivity.

Some users do not want an API client with a non-expiring Refresh Token for security reasons. We should highlight the difference instead of stating this as a requirement.

**Comment by bryanyap** [comment]
@eeshansim Makes sense to me. But, it would be good to let the customer know even as they are creating the connections. Information in the docs might get too buried and difficult to find.
